frontend/e2e/pages/metrics/config-step.ts

```diff
@@ -156,6 +160,10 @@ export class ConfigStep {
     this.sourceControlTokenErrorMessage = this.sourceControlContainer.getByText('Token is incorrect!');
 
     this.saveAsButton = page.getByRole('button', { name: 'Save' });
+    this.addNewTimeRangeButton = page.getByRole('button', { name: 'Button for adding date range' });
+    this.errorStartTimeMessage = page.getByText('Start date is invalid');
+    this.errorEndTimeMessage = page.getByText('End date is invalid');
+    this.removeTimeRangeButtons = page.getByText('Remove');
```

Comment:
Since the `Remove` is a quite common word, suggest to select by `basicInfoContainer.getByText` to avoid potential confilicts with any other remove button from other modules(even there is no other remove button now)

Reply:
done